atom/app/atom_content_client.cc

```diff
@@ -27,4 +64,19 @@ void AtomContentClient::AddAdditionalSchemes(
   standard_schemes->push_back("chrome-extension");
 }
 
+void AtomContentClient::AddPepperPlugins(
+  std::vector<content::PepperPluginInfo>* plugins) {
+  auto command_line = base::CommandLine::ForCurrentProcess();
+  auto flash_path = command_line->GetSwitchValueNative(
+      switches::kPpapiFlashPath);
+  if (flash_path.empty())
+    return;
+
+  auto flash_version = command_line->GetSwitchValueASCII(
+      switches::kPpapiFlashVersion);
```

Comment:
its optional info for the plugin used only for url request headers to [check](https://code.google.com/p/chromium/codesearch#chromium/src/content/renderer/pepper/url_request_info_util.cc&cl=GROK&l=161&rcl=1431214825). But if users are loading multiple flash plugin at the same time, they should specify anyway. Should that be strict criteria ?